packages/picasso/src/InputLimitAdornment/InputLimitAdornment.tsx

```diff
@@ -70,7 +70,10 @@ const InputLimitAdornment = (props: Props) => {
         data-testid={testIds?.inputAdornment}
         error={error}
       >
-        {Math.abs(charsTillLimit)} {multilineLabel}
+        <span data-testid='message'>
```

Comment:
```suggestion
        <span data-testid={testIds?.message}>
```

we should not hard code the test id

Reply:
:white_check_mark: